src/signals-mach.c

```diff
@@ -279,12 +279,16 @@ kern_return_t catch_mach_exception_raise(
     int nthreads = jl_atomic_load_acquire(&jl_n_threads);
     for (tid = 0; tid < nthreads; tid++) {
         jl_ptls_t _ptls2 = jl_atomic_load_relaxed(&jl_all_tls_states)[tid];
+        if (_ptls2->current_task == NULL) {
```

Comment:
```suggestion
        if (jl_atomic_load_relaxed(&_ptls2->current_task) == NULL) {
```

The `current_task` field is _Atomic, so we have a verifier rule that you must specify the intended ordering for it. For this field, the only defined ordering for a load is usually `relaxed`, since we don't have any other ordering defined when we release stored it. (And relaxed is what `jl_thread_suspend_and_get_state2` uses for the same check).

Reply:
Thanks

Reply:
Why didn't the verifier catch the unspecified load that was previously below?

Reply:
I think we skip this file in the verifier, since we only run it on linux for CI.